src/icons/imported/Icon/IconBoxApp512.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+/* eslint-disable react/jsx-sort-props */
```

Comment:
Is this necessary?

Reply:
Added this to account for the props in the raw SVGs not being in any specific order. From design's (and their tool's) standpoint, I don't think they have as much control over the order of SVG attributes. And one of the goals in the importing was to keep as much the same as possible, so I introduced this exception.

Related question to me is whether we should annotate these as "auto-generated" somehow? so that people don't go in and tweak them.

Reply:
Is this eslint-disable still needed with the external type declaration?

Reply:
Yah, currently we do not have a way to coerce the ordering of the props in the exported SVGs, and this rule is complaining about "width" coming before "height" in the converted `AccessibleSVG` prop list.